docs/reference/setup/install.asciidoc

```diff
@@ -27,6 +27,12 @@ Elasticsearch website or from our RPM repository.
 +
 <<rpm>>
 
+`docker`::
+
+An image is available for running Elasticsearch as a Docker container. It ships with https://www.elastic.co/guide/en/x-pack/current/index.html[X-Pack] pre-installed and may be downloaded from the Elastic Docker Registry.
```

Comment:
Make the link to x-pack a `:variable:` in index.asciidoc, so we can update version links easily.  Use `5.0` instead of `current`